config/jobs/kubernetes/generated/generated.yaml

```diff
@@ -20,15 +20,12 @@ periodics:
       - --provider=gce
       - --gcp-zone=us-west1-b
       - --gcp-node-image=gci
-      - --extract=ci/latest-1.29
+      - --extract=ci/latest-1.30
       - --extract-ci-bucket=k8s-release-dev
       - --timeout=180m
       - --env=KUBE_PROXY_DAEMONSET=true
       - --env=ENABLE_POD_PRIORITY=true
-      # TODO: Fix generate_tests.py to set DisableCloudProviders,DisableKubeletCloudCredentialProviders to true when --cloud-proider=external
-      # And also add --env=ENABLE_AUTH_PROVIDER_GCP=true
-      - --env=KUBE_FEATURE_GATES=AllAlpha=true,InTreePluginGCEUnregister=false,DisableCloudProviders=true,DisableKubeletCloudCredentialProviders=true
-      - --env=ENABLE_AUTH_PROVIDER_GCP=true
+      - --env=KUBE_FEATURE_GATES=AllAlpha=true,InTreePluginGCEUnregister=false,DisableCloudProviders=false,DisableKubeletCloudCredentialProviders=false
```

Comment:
@dims I have lost the context on this. But I found: https://github.com/kubernetes/kubernetes/pull/117503, can you please confirm if this is fine?

Reply:
yes this is fine.